test/common/http/http3/BUILD

```diff
@@ -0,0 +1,34 @@
+load(
+"//bazel:envoy_build_system.bzl",
+"envoy_cc_test",
+"envoy_package",
+"envoy_select_enable_http3",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+envoy_cc_test(
+    name = "conn_pool_test",
+    srcs = envoy_select_enable_http3(["conn_pool_test.cc"]),
+    tags = ["nofips"],
```

Comment:
nofips? As in the compliance standard?

Reply:
yeah.  they did a fips version of boring a year or two back but we need a more modern version of boringssl for some of the QUIC crypto functions, so our QUIC impl is not compatible with Envoy builds latched to the fips boring.